serialized-lob/README.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: pattern
+title: Serialized LOB
+folder: serialized-lob
+permalink: /patterns/serialized-lob/
+categories: Creational
+language: en
+tags:
+- Data access
+---
```

Comment:
See the yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute